hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -623,11 +623,44 @@ public Response pullOverpassToDb(@Context HttpServletRequest request,
         // Write the data to the hoot db
         GrailParams params = new GrailParams();
         params.setUser(user);
+        /**
+         * Readable format of the below url:
+         *
+         * [out:json][timeout:3600][bbox:{{bbox}}];
+         * (
+         *   node;<;>;
+         * )->.all;
+         *
+         * (
+         *   relation[natural=coastline];
+         *   way[natural=coastline];<;>;
+         *   relation[boundary=census];>;
+         *   relation[boundary=administrative];
+         *   way[boundary=administrative];>;
+         *   node[boundary=administrative];
+         * )->.exclude;
+         *
+         * (.all; - .exclude;);
+         *
+         * out body;
+         * >;
+         * out meta qt;
+         *
+         */
         String url = "'"
                 + PUBLIC_OVERPASS_URL
-                + "/api/interpreter?data=[out:json];(node("
-                + new BoundingBox(bbox).toOverpassString()
-                + ");<;>;);out%20meta%20qt;"
+                + "/api/interpreter?data=[out:json][bbox:" + new BoundingBox(bbox).toOverpassString() + "];"
+                + "(node;<;>;)->.all;"
+                + "(relation[natural=coastline];"
+                + "way[natural=coastline];<;>;"
+                + "relation[boundary=census];>;"
+                + "relation[boundary=administrative];"
+                + "way[boundary=administrative];>;"
+                + "node[boundary=administrative];"
+                + ")->.exclude;"
+                + "(.all;%20-%20.exclude;);"
+                + "out%20body;>;"
+                + "out%20meta%20qt;"
```

Comment:
This may be messy, but do we want to consider moving this query to a config file so it could be modified outside of a new Hoot release?